src/ILLink.RoslynAnalyzer/ISymbolExtensions.cs

```diff
@@ -95,7 +95,7 @@ public static string GetDisplayName (this ISymbol symbol)
 			var sb = new StringBuilder ();
 			switch (symbol) {
 			case IFieldSymbol fieldSymbol:
-				sb.Append (fieldSymbol.ContainingSymbol.ToDisplayString (ILLinkTypeDisplayFormat));
+				sb.Append (fieldSymbol.ContainingSymbol?.ToDisplayString (ILLinkTypeDisplayFormat));
```

Comment:
I'd prefer not to add `?` unless we actually have some kind of reasonable handling for the null case, and this would produce a weird-looking string if the containing symbol is null. Do we know of a way that a field symbol can have a null ContainingSymbol?

I'd be fine with just leaving out the `Append (".")` in that case. Similar for the cases below.

Reply:
Good point, most of those cases we can assume the ContainingSymbol/ContainingType will not be null.